openssl-sys/build.rs

```diff
@@ -464,14 +467,22 @@ See rust-openssl README for more information:
         println!("cargo:libressl_version=26x");
         println!("cargo:version=101");
         Version::Libressl
+    } else if expanded.contains("RUST_OPENSSL_111") {
+        println!("cargo:rustc-cfg=ossl11x");
+        println!("cargo:rustc-cfg=ossl111");
+        println!("cargo:version_major=11x");
```

Comment:
Let's not output version_major for now.

Reply:
This was added so that systest's build.rs can set the proper `cfg`s. Is there a different approach I should use?

Reply:
You can have systest's build.rs parse version.